File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/Student.java
package com.bnuz.oke.entity;

/**
 * Student
 *
 * @author handsome
 * @date 2020年 05月15日 00:00:33
 */
public class Student {

	private int studentId;

	private String studentNumber;

	private String studentName;

	private String studentClass;

	private User user;

	public int getStudentId() {
		return studentId;
	}

	public void setStudentId(int studentId) {
		this.studentId = studentId;
	}

	public String getStudentNumber() {
		return studentNumber;
	}

	public void setStudentNumber(String studentNumber) {
		this.studentNumber = studentNumber;
	}

	public String getStudentName() {
		return studentName;
	}

	public void setStudentName(String studentName) {
		this.studentName = studentName;
	}

	public String getStudentClass() {
		return studentClass;
	}

	public void setStudentClass(String studentClass) {
		this.studentClass = studentClass;
	}

	public User getUser() {
		return user;
	}

	public void setUser(User user) {
		this.user = user;
	}

	@Override
	public String toString() {
		return "Student{" +
				"studentId=" + studentId +
				", studentNumber='" + studentNumber + '\'' +
				", studentName='" + studentName + '\'' +
				", studentClass='" + studentClass + '\'' +
				", user=" + user +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Entity/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 学生信息封住类
    /// </summary>
    [Serializable]
    public class Student
    {
        //学生id
        public int studentId { set; get; }
        //学号
        public string studentNumber { set; get; }
        //学生姓名
        public string studentName { set; get; }
        //学生班级
        public string studentClass { set; get; }
        //学生账号
        public User user { set; get; }
    }
}
